src/components/tables/AccessGrants/AccessLinks/Dialog/GenerateInvitation.tsx

```diff
@@ -21,12 +32,59 @@ import { appendWithForwardSlash, hasLength } from 'src/utils/misc-utils';
 // outside of advanced cases.
 const capabilityOptions = ['admin', 'read'];
 const typeOptions = ['single-use', 'multi-use'];
+const MAX_PREFIX_LENGTH = 12;
+
+const RadioOption = ({
+    value,
+    label,
+    isSelected,
+    onClick,
+}: {
+    value: string;
+    label: ReactNode;
+    isSelected: boolean;
+    onClick?: () => void;
+}) => {
+    return (
+        <FormControlLabel
+            value={value}
+            control={
+                <Radio
+                    size="small"
+                    sx={{
+                        '&.MuiRadio-root:hover': {
+                            bgcolor: 'transparent !important',
```

Comment:
I would just leave this. It does look a bit wonky but we leave these default states in most spots. Also, this will still show with keyboard controls and we _generally_ try to keep keyboard and mouse interactions pretty close.

Example is the table below this form:
<img width="193" height="137" alt="Image" src="https://github.com/user-attachments/assets/8fd8be8c-a21f-4810-96f6-db0658db759a" />